src/Marketplace/MarketplaceScreen.js

```diff
@@ -22,9 +30,21 @@ const styles = StyleSheet.create({
     marginTop: 10,
     backgroundColor: '#0492c2',
   },
+  card: {
+    width: '90%',
+  },
 });
 
 export default function MarketplaceScreen() {
+  const [card, setCard] = useState([]);
+
+  useEffect(() => {
+    base('APIFM Produce Cards').find('recuOS5skQ04lUxBk', (err, record) => {
+      if (err) { console.error(err); return; }
```

Comment:
Can you replace the console log with an alert or some onscreen popup? We typically don't want to be logging to the console unless absolutely necessary.